src/pages/ThorChainLP/components/LpType.tsx

```diff
@@ -139,12 +181,105 @@ export const LpType = ({
     setValue(fromOpportunityId(opportunityId).opportunityType)
   }, [opportunityId, setValue])
 
+  const subtitleStyle = useMemo(() => {
+    return {
+      '& svg': {
+        display: 'inline-block',
+        verticalAlign: 'middle',
+      },
+    }
+  }, [])
+
+  const informationsByPosition = useMemo(() => {
+    const displayAmounts = (position: AsymSide | 'sym') => ({
+      text: amountsByPosition?.[position] ? (
+        <>
+          <Amount.Crypto
+            value={amountsByPosition?.[position].underlyingAssetAmountCryptoPrecision}
+            symbol={asset?.symbol ?? ''}
+            maximumFractionDigits={6}
+          />
+          <Amount.Crypto
+            value={amountsByPosition?.[position].underlyingRuneAmountCryptoPrecision}
+            symbol={thorchainAsset?.symbol ?? ''}
+            maximumFractionDigits={6}
+          />
+        </>
+      ) : (
+        <>
+          <Skeleton mb={1} height='14px' width='100%' />
+          <Skeleton height='14px' width='100%' />
+        </>
+      ),
+      props: {
+        mt: !amountsByPosition?.[position] ? 2 : 1,
+      },
+    })
+
+    const newPosition = {
+      text: translate('pools.newPosition'),
+      icon: <CircleCrossIcon />,
+      tooltipText: translate('pools.newPositionTooltip'),
+    }
+    const notSupported = {
+      text: translate('pools.notSupported'),
+      icon: <CircleXIcon />,
+      props: { color: 'text.error' },
+      tooltipText: translate('pools.notSupportedTooltip'),
+    }
+
+    const runeInformations: PositionInformations = (() => {
+      if (hasSymPosition) return displayAmounts('sym')
+      if (hasAsymAssetPosition) {
+        return hasAsymRunePosition ? displayAmounts(AsymSide.Rune) : newPosition
+      }
+      if (hasAsymRunePosition) return displayAmounts(AsymSide.Rune)
+
+      return newPosition
+    })()
+
+    const assetInformations: PositionInformations = (() => {
+      if (hasAsymRunePosition || hasSymPosition) {
+        return hasAsymAssetPosition ? displayAmounts(AsymSide.Asset) : newPosition
+      }
+      return hasAsymAssetPosition ? displayAmounts(AsymSide.Asset) : newPosition
+    })()
+
+    const symInformations: PositionInformations = (() => {
+      if (hasAsymRunePosition) return notSupported
+      if (hasAsymAssetPosition) {
+        return hasSymPosition ? displayAmounts('sym') : newPosition
+      }
+      if (hasSymPosition) return displayAmounts('sym')
+
+      return newPosition
+    })()
```

Comment:
Also quite confusing. Perhaps:

```suggestion
    const symInformations: PositionInformations = (() => {
      if (hasAsymRunePosition) return notSupported
      if (hasSymPosition) return displayAmounts('sym')
      return newPosition
    })()
```

Reply:
Changed in [e22aec7](https://github.com/shapeshift/web/pull/7367/commits/e22aec780b3128ece64322e6a640b9af9d997090)